plone/app/vocabularies/types.py

```diff
@@ -8,6 +8,7 @@
 from zope.schema.interfaces import IVocabularyFactory
 from zope.schema.vocabulary import SimpleTerm
 from zope.schema.vocabulary import SimpleVocabulary
+from unidecode import unidecode
```

Comment:
Add `unidecode` as a dependency in [setup.py](https://github.com/plone/plone.app.vocabularies/blob/a7b0049757f15694fee209b2701bab08a476803a/setup.py#L45)

Run `tox -e lint` to check for lint errors.

@stevepiercy the link in

https://github.com/plone/plone.app.vocabularies/blob/master/CONTRIBUTING.rst

is broken.

Reply:
> Add unidecode as a dependency in [setup.py](https://github.com/plone/plone.app.vocabularies/blob/a7b0049757f15694fee209b2701bab08a476803a/setup.py#L45)

added

Reply:
In alphabetical order please.

Reply:
> In alphabetical order please.

done